scripts/generate_openapi_document.py

```diff
@@ -0,0 +1,34 @@
+import json
+import os
+from unittest.mock import (
+    patch,
+)
+
+from azul import (
+    config,
+)
+from azul.files import (
+    write_file_atomically,
+)
+from azul.modules import (
+    load_app_module,
+)
+
+
+def main():
+    with patch.dict(os.environ,
```

Comment:
I think I asked this before but don't remember the answer. Why do we patch the environment instead of the config here?